static/js/browser2/out_arcs.tsx

```diff
@@ -0,0 +1,157 @@
+      if (!ignoredOutArcProperties.has(label)) {
+        return axios
+          .get(`/api/browser/propvals/${label}/${this.props.dcid}`)
+          .then((resp) => resp.data);
+      }
+    });
+    Promise.all(propValuesPromises).then((propValuesData) => {
+      const outArcsByPredicate = {};
+      const propertyLabels = [];
+      propertyLabels.push("typeOf");
+      propValuesData.forEach((valuesData) => {
+        if (!valuesData || _.isEmpty(valuesData.values)) {
+          return;
+        }
+        const predicate = valuesData.property;
+        let values = valuesData.values.out;
+        if (predicate === "nameWithLanguage") {
+          const extra = values.length - maxNameWithLanguageValues;
+          values = _.slice(values, 0, maxNameWithLanguageValues);
+          values.push({
+            value: "(... " + extra + " more ...)",
+          });
+        }
+        outArcsByPredicate[predicate] = values;
+        if (predicate !== "typeOf") {
+          propertyLabels.push(predicate);
+        }
+      });
+      outArcsByPredicate["dcid"] = [{ value: this.props.dcid }];
+      propertyLabels.push("dcid");
+      this.setState({
+        arcValuesData: outArcsByPredicate,
+        propertyLabels: propertyLabels,
+      });
+    });
+  }
+
+  render(): JSX.Element {
+    if (_.isEmpty(this.state.propertyLabels)) {
+      return null;
+    }
+    return (
+      <div>
+        <table className="node-table">
+          <tbody>
+            <tr key="header">
+              <td className="property-column" width="25%">
+                <strong>Property</strong>
+              </td>
+              <td width="50%">
+                <strong>Value</strong>
+              </td>
+              <td width="25%">
+                <strong>Provenance</strong>
+              </td>
+            </tr>
+            {this.state.propertyLabels.map((propertyLabel) => {
+              const valuesArray =
+                propertyLabel in this.state.arcValuesData
+                  ? this.state.arcValuesData[propertyLabel]
+                  : [];
+              return valuesArray.map((value, index) => {
+                return (
+                  <tr key={propertyLabel + index}>
+                    <td className="property-column" width="25%">
+                      <a href={"/browser/browser2/" + propertyLabel}>
+                        {propertyLabel}
+                      </a>
+                    </td>
+                    <td width="50%">
+                      {value.dcid ? (
+                        <a href={"/browser/browser2/" + value.dcid}>
+                          {value.name ? value.name : value.dcid}
+                        </a>
+                      ) : (
+                        <span className="out-arc-text">{value.value}</span>
+                      )}
+                    </td>
+                    <td width="25%">
+                      {value.provenanceId ? (
```

Comment:
you can also just do:
(value.provenanceId && <a>...</a>)

Reply:
done